src/VisualStudio/CSharp/Impl/CodeModel/CSharpCodeModelService.cs

```diff
@@ -1140,7 +1140,8 @@ private EnvDTE.vsCMAccess GetDefaultAccessibility(SyntaxNode node)
                 }
             }
 
-            if (node is AccessorDeclarationSyntax)
+            if (node is AccessorDeclarationSyntax || 
+                node is ArrowExpressionClauseSyntax)
             {
                 return GetAccess(node.FirstAncestorOrSelf<BasePropertyDeclarationSyntax>());
```

Comment:
it was actually the getter that was throwing the issue in CodeFunction.Access so no normal accessors where fine.